app/waf/src/models/service.model.ts

```diff
@@ -301,8 +333,12 @@ export class Service extends CommonEntity {
     type: 'string',
     required: false,
     schema: {
-      create: false,
-      update: false,
+      create: true,
+      update: true,
+      response: true,
+    },
+    as3: {
+      type: 'bigip',
```

Comment:
use or bigip?

Reply:
in profileHTTP support PR, it will be changed to `extends`. 
remove it here.